LWOTC_WeaponPack/Localization/Coils/XComGame.int

```diff
@@ -88,6 +88,13 @@ BriefSummary="Should their power fail them for some reason, the Templars rely on
 TacticalText="<Bullet/> The Autopistol gives the Templar the flexibility to combat the enemy in a conventional manner when not using their Psionic abilities.\n<Bullet/> Autopistols can be fired with a single action."
 AbilityDescName="autopistol"
 
+[SparkRifle_CG X2WeaponTemplate]
+FriendlyName="Coil Spark Rifle"
+FriendlyNamePlural="Coil Spark Rifles"
+BriefSummary=""
+TacticalText=""

```

Comment:
Can you copy paste the conventional/mag/beam text? You have creative freedom here.

Reply:
I just bought the spark DLC actually, I think that the text is like missing from my setup. I can't find it either way.

Reply:
Haha okay, I'll go in and make some new localization text for coil and laser spark rifles

Reply:
Honestly though, I swear I can't find the base game localization text for the guns. I've searched everything.